test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -1,15 +1,31 @@
 package seedu.addressbook.common;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import org.junit.Test;
 
 import java.util.Arrays;
-import java.util.List;
 
-import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNullTest() {
+        Utils testing = new Utils();
+
```

Comment:
Creating an object for the Utils class is not necessary here. You can access the method by using classname.method. e.g. Utils.isAnyNullTest()